articles/personal_growth.md

```diff
@@ -0,0 +1,30 @@
+---
+authorName: Sabrina Lange
+authorGithubUsername: sweinzie 
+issue: 97
+title: Why every Project Should Have Gardeners
+---
+# {{page.title}}
+
+As a gardner ist taking care of his plants to bring some fruit, we should also have people around in the projects helping us to bring learn and develop ourself to the next level. Unfortunatelly this is no common practice yet, therefore i would like to show you how this may work within our projects by keeping the picture of a gardener.
```

Comment:
"helping us to _bring_ learn and develop ourself to the next Level" -> "helping us to learn and develop ourself to the next level" ?